### PR TITLE
Change to titles for the "create fund" journey

### PR DESCRIPTION
**Pull request by duffylouise**
Changed 7 titles from statements to questions based on the external content review

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [ ] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by lozette** [comment]
I am going to hold off fixing the tests on this one until #409 is merged as that changes some of the labelling/hint texts

**Comment by duffylouise** [comment]
No problem. There might be a couple more PRs coming your way this afternoon.

File: spec/features/staff/users_can_choose_recipient_country_spec.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     context "with JavaScript disabled" do
       scenario "countries are choosen from a select box" do
-        expect(page).to have_select(I18n.t("page_title.activity_form.show.country"))
```
Comment (tahb) [inline spec/features/staff/users_can_choose_recipient_country_spec.rb:17]: I wasn't expecting page_title to be affected by any changes we made to the form builder gem. The `@page_title` is set in the `ActivityFormController` and then passed to the `_wrapper` on each form step. This allows us to keep our form titles different from individual field names.

Having a quick look I think we've accidentally broken page titles for country prior to this work?

Our en.yml says:

```
  page_title:
    activity_form:
      show:
        country: What country will benefit from this activity?
```

But the form view on staging just has "country" and I think we want the former which would make these switches to activerecord.etc no longer needed!
Screenshot 2020-05-29 at 10 58 06

Page titles seem to be working for other fields like identifier and sector.

File: spec/features/staff/users_can_create_a_fund_level_activity_spec.rb [hunk 2]
```diff
@@ -141,7 +141,7 @@
 
         choose "Primary education"
         click_button I18n.t("form.activity.submit")
-        expect(page).to have_content I18n.t("page_title.activity_form.show.status")
+        expect(page).to have_content I18n.t("activerecord.attributes.activity.status")
```
Comment (tahb) [inline spec/features/staff/users_can_create_a_fund_level_activity_spec.rb:144]: If the page title for status is staying as "Status" then I'd be looking to see why our pape_title code in the wrapper stopped working. If we stop using page title we lose the ability to change this value to something other than the active record attribute name, which removes the previous flexibility.

**Comment by lozette** [comment]
I'm going to close this PR and re-start it fresh

**Comment by duffylouise** [comment]
Good thinking - thanks.

**Pull request closed without merging**